react-components/stories/Image360CollectionContainer.stories.tsx

```diff
@@ -37,7 +37,7 @@ export const Main: Story = {
 };
 
 type CadModelContainerStoryContentProps = {
-  addImageCollection360Options: AddImageCollection360Options;
+  addImageCollection360Options: AddImage360CollectionOptions;
```

Comment:
same here: If it doesnt break anything, maybe it is also good to update the parameter's name to follow the same pattern.